docs/guide/use_cases/self_managed_lb/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Self Managed Load Balancers
+---
+
+## Motivation
+
+The load balancer controller (LBC) generally creates and destroys [AWS Load Balancers](https://docs.aws.amazon.com/elasticloadbalancing/index.html) in response to Kubernetes resources. 
+
+However, some cluster operators may prefer to manually manage AWS Load Balancers. This supports use cases like:
+- Preventing acciential release of key IP addresses.
+- Supporting load balancers where the Kubernetes cluster is one of multiple targets.
+- Complying with organizational requirements on provisioning load balancers, for security or cost reasons. 
+
+## Solution Overview
+
+Use the TargetGroupBinding CRD to sync a Kubernetes service with the targets of a load balancer. 
+
+First, a load balancer is manually created directly with AWS. A [listener](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-listeners.html) and a [target group](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html) is added to the load balancer. 
```

Comment:
The example below uses NLB. Lets be consistent.

Lets also use active voice for this paragraph.